emcache/client.py

```diff
@@ -322,7 +326,12 @@ async def get(self, key: bytes, return_flags=False) -> Optional[Item]:
             else:
                 return await self._autobatching_flags_nocas.execute(key)
 
-        keys, values, flags, _ = await self._fetch_command(b"get", key)
+        result = await self._fetch_command(b"get", key)
```

Comment:
If we use the `set_exception` as proposed before we can keep this as it was, so the ```await self._fetch_command(b"get", key)``` in case of an error will just raise a `CommandError` Exception

Reply:
await Error raise [in extractor parser with skip handling and raise TimeoutError](https://github.com/emcache/emcache/pull/146/files#diff-07f845da2abf106971aad8cdbb60c413d2cc88fe54d91bb4cebb28b3e4c43de4R173)